packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -8,6 +8,8 @@ import type { Capabilities, Services, Options } from '@wdio/types'
 import { BrowserstackConfig } from './types'
 
 const log = logger('@wdio/browserstack-service')
+// @ts-ignore
+import { bstackServiceVersion } from '../package.json'
```

Comment:
What is `bstackServiceVersion` suppose to be? Are you referring to `version`, maybe this is what you wanted:
```suggestion
// @ts-ignore
import { version as bstackServiceVersion } from '../package.json'
```

Reply:
yes, will fix this and add test.